new-content/events/summer-school-2021-page/faq.md

```diff
@@ -7,40 +7,56 @@ module: faq
 title: Frequently asked questions
 khanacademy:
   url: https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro
-  segment: khanacademy-matrix-multiplication-intro
+  segment:
+      action: 'events-summer-school > faq > khanacademy-matrix-multiplication-intro'
 pythonandjupyter:
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
-  segment: qiskit-python-and-jupyter-notebooks
+  segment:
+      action: 'events-summer-school > faq > qiskit-python-and-jupyter-notebooks'
 linearalgebra:
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
```

Comment:
The link says "linear algebra" but it's pointing to the jupyter notebooks. Also, I think the linear algebra section was removed from the textbook. could you check this and change the url in consequence please?